docs/source/developer_guides/lora.md

```diff
@@ -201,6 +165,28 @@ Assuming the original model had 5 layers `[0, 1, 2 ,3, 4]`, this would create a
 [Fewshot-Metamath-OrcaVicuna-Mistral-10B](https://huggingface.co/abacusai/Fewshot-Metamath-OrcaVicuna-Mistral-10B) is an example of a model trained using this method on Mistral-7B expanded to 10B. The
 [adapter_config.json](https://huggingface.co/abacusai/Fewshot-Metamath-OrcaVicuna-Mistral-10B/blob/main/adapter_config.json) shows a sample LoRA adapter config applying this method for fine-tuning.
 
+## Optimizers
+
+LoRA training can optionally include special purpose optimizers. Currently the only such optimizer is LoRA+.
+
+### LoRA+ optimized LoRA
+
+LoRA training can be optimized using [LoRA+](https://arxiv.org/abs/2402.12354), which uses different learning rates for the adapter matrices A and B, shown to increase finetuning speed by up to 2x and performance by 1-2%.
+
+```py
+from transformers import Trainer
+from peft.optimizers import create_loraplus_optimizer
+
+optimizer = create_loraplus_optimizer(...)
```

Comment:
Thanks for updating the docs.

In my example, the `...` were just intended as placeholders. At least for `create_loraplus_optimizer`, it would be good to show an actually valid call. Let's also include the part where the LoRA model is initialized, otherwise users may get the impression that this will work with non-LoRA models.

Reply:
By `the part where the LoRA model is initialized` are you referring to `get_peft_model()` or something else?

I pushed a commit that adds params to the optimizer.

Reply:
> By `the part where the LoRA model is initialized` are you referring to `get_peft_model()`

Yes, to make it crystal clear that a LoRA model is expected here.

Reply:
Right. I restored assumption and made it specifically say LoRA. If this isn't what you had in mind I'll think some more and rewrite it tomorrow. :)